go-controller/pkg/ovn/ovn.go

```diff
@@ -134,8 +134,8 @@ func (oc *Controller) addDefaultLogicalPort(pod *kapi.Pod) {
 }
 
 func (oc *Controller) addLogicalPortsToExtraSwitches(pod *kapi.Pod) {
-	switches := oc.getNetworkNamesFromPodAnnotations(pod.Annotations)
```

Comment:
generally, in multus, so we allow pods to use CRDs only from their own namespace?

Reply:
Seems like the network can have different CRD then the pod. I will fix the code to do something similar to  https://github.com/intel/multus-cni/blob/67e661f932a6aba869275a0a4d7db865fea888ae/k8sclient/k8sclient.go#L192